src/main/java/org/apache/maven/plugins/checkstyle/exec/DefaultCheckstyleExecutor.java

```diff
@@ -51,20 +51,23 @@
 import org.apache.maven.project.MavenProject;
 import org.codehaus.plexus.component.annotations.Component;
 import org.codehaus.plexus.component.annotations.Requirement;
-import org.codehaus.plexus.logging.AbstractLogEnabled;
 import org.codehaus.plexus.resource.ResourceManager;
 import org.codehaus.plexus.resource.loader.FileResourceCreationException;
 import org.codehaus.plexus.resource.loader.FileResourceLoader;
 import org.codehaus.plexus.resource.loader.ResourceNotFoundException;
 import org.codehaus.plexus.util.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * @author Olivier Lamy
  * @since 2.5
  *
  */
 @Component(role = CheckstyleExecutor.class, hint = "default", instantiationStrategy = "per-lookup")
-public class DefaultCheckstyleExecutor extends AbstractLogEnabled implements CheckstyleExecutor {
+public class DefaultCheckstyleExecutor implements CheckstyleExecutor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultCheckstyleExecutor.class);
```

Comment:
LOGGER is not a named constant and should be logger instead of LOGGER. See https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

Reply:
Ah I had changed that in 0ee70f2b235015b7fb85c7555eed54a5c07fe9a8 after Checkstyle complained locally; makes sense not to use uppercase, but then we might need to change checkstyle.

Reply:
Thanks for the review! Picked up in https://github.com/apache/maven-checkstyle-plugin/pull/157/commits/bc1665009162135424ea9f4d4a0a31ce9ef351ea